final-project/manRoomStates.tex

```diff
@@ -0,0 +1,21 @@
+\chapter{Maintaining Room State \\
+  \small{\textit{-- Evan Ciok, Sophia DiCuffa, Carson McManus}}
+  \index{Chapter!room-state}
+  \label{Chapter::RoomState}}
+
+\section{Lost Balancer-Monolith Connections}
+
+The connection between the load balancer and the individual monoliths is vital to the operation of OTT, so when one of these connections is lost, it needs to be handled appropriately. 
+The most important consideration is what to do with all clients connected to rooms on that given monolith. Without the balancer connecting to it, it is no longer viable and all clients 
+should be unloaded from the rooms. From there, they can reconnect to a room through the load balancer to a monolith with a valid connection. 
```

Comment:
A few points here:
- The monolith always initiates the websocket connection to the balancer. The balancer only accepts inbound websocket connections, and does not make any outgoing websocket connections.
- clients should be _removed_ from rooms, not _unloaded_
- the clients connected to the balancer need to be disconnected
- I feel like you're padding out the important information with more words than is necessary. For example:

> The connection between the load balancer and the individual monoliths is vital to the operation of OTT, so when one of these connections is lost, it needs to be handled appropriately.

This is overly verbose for the amount of information it conveys, and the first half is obvious at this point in the paper. You could say

> When a Balancer-Monolith connection is lost, clients connected to that monolith need to be handled appropriately. Otherwise, the accuracy of room state is compromised.